python/dask_cudf/dask_cudf/core.py

```diff
@@ -263,6 +263,12 @@ def drop(df, keep_keys):
         results = [p for i, p in enumerate(parts) if uniques[i]]
         return from_delayed(results, meta=self._meta).reset_index()
 
+    def to_parquet(self, path, *args, **kwargs):
+        """ Calls dask.dataframe.io.to_parquet with CudfEngine backend """
+        from dask_cudf.io import to_parquet
+
+        to_parquet(self, path, *args, **kwargs)
```

Comment:
```suggestion
        return to_parquet(self, path, *args, **kwargs)
```

I think that we might return filenames in some cases?  Even if we don't do this now this should future-proof us.